CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## [Unreleased]
 
+### Added
+
+-
+
+### Fixed
+
+- `get_mlflow_config` now uses the kedro context config_loader to get configs (#66 #64 #54 #30 #31 #72)
+
+### Changed
+
+- `MlflowNodeHook` have now a before_pipeline_run hook allowing to load the kedro context, get configs and populating MlflowNodeHook attributes
```

Comment:
[...] a `before_pipeline_run` hook which stores the  `ProjectContext` and enable to retrieve configuration.